build_debian.sh

```diff
@@ -176,6 +176,11 @@ sudo chmod +x $FILESYSTEM_ROOT/etc/initramfs-tools/scripts/init-bottom/varlog
 sudo cp files/initramfs-tools/union-fsck $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/union-fsck
 sudo chmod +x $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/union-fsck
 pushd $FILESYSTEM_ROOT/usr/share/initramfs-tools/scripts/init-bottom && sudo patch -p1 < $OLDPWD/files/initramfs-tools/udev.patch; popd
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+    sudo cp files/initramfs-tools/arm-utils $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/arm-utils
+    sudo chmod +x $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/arm-utils
```

Comment:
i think we need to differentiate the arm and the bootloader. this is not really related to arm but bootloader. I do not mind to install then in x86 build as long as they are not too large.

Reply:
Ok will remove and change name to uboot-utils